pkg/manifests/manifests.go

```diff
@@ -1043,11 +1048,19 @@ func (f *Factory) ThanosQuerierRoute() (*routev1.Route, error) {
 	return r, nil
 }
 
+func (f *Factory) SharingConfigDeprecated(promHost, amHost, grafanaHost, thanosHost *url.URL) *v1.ConfigMap {
```

Comment:
let's add a unit test in `manifests_test.go`

Reply:
done